viewPost.php

```diff
@@ -0,0 +1,7 @@
+<?php
+include('config/init.php');
+    headerA("Getting to Know Me", "My Blog!");
+    $postId = $_REQUEST['postid'];
+    blog($postId);
```

Comment:
This relates to a comment I left above. I suggested that you don't reference `$_REQUEST` directly inside a function. You're one step ahead of me by passing `$postId` here, but you're not actually using it in that function.